### PR TITLE
Task3/database

### PR DESCRIPTION
**Pull request by Frazeny**

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -4,22 +4,29 @@ import { errorHandlerMiddleware } from './middleware/error-handler'
...
 // middleware
+app.use(express.json())
 app.use(notFoundMiddleware)
 app.use(errorHandlerMiddleware)
 
 const start = async (): Promise<void> => {
   try {
+    await connectDB(
+      process.env.MONGO_URL ??
+        'mongodb+srv://vitaliivborshchov:12345@epam-nodejs-course-prac.gxfdvhd.mongodb.net/MoviesDB?retryWrites=true&w=majority'
```
Comment (vasylkivskyy) [inline src/main.ts]: Do not push sensitive data to github.

File: src/middleware/error-handler.ts [hunk 2]
```diff
@@ -1,9 +1,19 @@
 import CustomAPIError from '../errors/custom-api-error'
 import { type Request, type Response, type NextFunction, type ErrorRequestHandler } from 'express'
```
Comment (vasylkivskyy) [inline src/middleware/error-handler.ts:2]: It is not necessary to add "type" in imports

**Review by vasylkivskyy — CHANGES_REQUESTED** [review]

File: src/main.ts (hunk 1 above)
Comment (Frazeny) [reply src/main.ts]: Yes, I agree. I included this dummy db temporarily solely for quick testing purposes to ensure smooth functionality😅

File: src/middleware/error-handler.ts (hunk 2 above)
Comment (Frazeny) [reply src/middleware/error-handler.ts:2]: Yes, it's just part of my eslint configuration, including @typescript-eslint/consistent-type-imports and @typescript-eslint/consistent-type-exports, and is placed automatically when npm lint is run. If needed, I can remove these rules😄

Comment (vasylkivskyy) [reply src/middleware/error-handler.ts:2]: Please, remove sensitive data and that's ok

Comment (Frazeny) [reply src/middleware/error-handler.ts:2]: Done! Sensitive data has been removed, and the database access has been changed. Let me know if there's anything else I should tweak

**Review by SergiiGEpam — APPROVED** [review]

**Pull request merged**